### PR TITLE
Introduce configuration options for integer, text and float questions

### PR DESCRIPTION
**Pull request by sliverc**
Validate answers whether those are correct according to the question type and configuration.

### CODE REVIEW

File: caluma/form/schema.py [hunk 1]
```diff
@@ -62,11 +66,29 @@ class Meta:
...
 
 class IntegerQuestion(graphene.ObjectType):
+    max_value = graphene.Int()
+    min_value = graphene.Int()
+
+    def resolve_max_value(self, info):
+        return self.configuration["max_value"]
+
+    def resolve_min_value(self, info):
+        return self.configuration["min_value"]
```
Comment (anehx) [inline caluma/form/schema.py]: You don't need those - delete 'em!

File: caluma/form/schema.py [hunk 2]
```diff
@@ -62,11 +66,29 @@ class Meta:
...
 
 class FloatQuestion(graphene.ObjectType):
+    min_value = graphene.Float()
+    max_value = graphene.Float()
+
+    def resolve_max_value(self, info):
+        return self.configuration["max_value"]
+
+    def resolve_min_value(self, info):
+        return self.configuration["min_value"]
```
Comment (anehx) [inline caluma/form/schema.py]: Same

File: caluma/form/schema.py (hunk 1 above)
Comment (sliverc) [reply caluma/form/schema.py]: True. Deleted. Also changed that max and min values are non null in query as we always have a default.

File: caluma/form/schema.py (hunk 2 above)
Comment (sliverc) [reply caluma/form/schema.py]: same as above

**Pull request merged**